packages/core/src/tracing/instrumentation/protocols/nativeFetch.js

```diff
@@ -80,19 +80,20 @@ function instrument() {
     // eslint-disable-next-line no-unused-vars
     let w3cTraceContext = cls.getW3cTraceContext();
 
-    const skipTracingResult = cls.skipExitTracing({ isActive, extendedResponse: true, skipParentSpanCheck: true });
-
-    // If there is no active entry span, we fall back to the reduced span of the most recent entry span. See comment in
-    // packages/core/src/tracing/clsHooked/unset.js#storeReducedSpan.
-    const parentSpan = cls.getCurrentSpan() || cls.getReducedSpan();
+    const skipTracingResult = cls.skipExitTracing({
+      isActive,
+      extendedResponse: true,
+      skipParentSpanCheck: true,
```

Comment:
issue: This looks wrong.

Before: we check the parent manually. Now: no one checks the parent.
Please break it with TDD.

Reply:
https://github.com/instana/nodejs/blob/bb263a8c4ca5186bfda7153b1442d4d0b5a62d64/packages/core/src/tracing/cls.js#L598

I have moved the basic check for parentSpan to `cls.skipExitTracing` function.
So there is no need to check in again in the instrumentation.